pvlib/modelchain.py

```diff
@@ -955,6 +960,19 @@ def no_extra_losses(self):
         self.losses = 1
         return self
 
+    def pvsyst_dc_losses(self):
```

Comment:
This is the only DC loss applied by pvsyst?  If no, is the plan to add to this method when other DC loss models are implemented?

Reply:
In PVsyst you can also input ohms directly, which we could support pretty easily I think with a different loss parameter that uses the same model chain function but bypasses the equivalent resistance function. I want to investigate that more to confirm the method and then will work to implement.